### PR TITLE
Rename some Peep member variables to use TitleCase

### PR DESCRIPTION
**Pull request by tupaschoal**
Continues #10653

It might be easier to review on a commit by commit basis

### CODE REVIEW

File: src/openrct2/peep/Peep.h [hunk 1]
```diff
@@ -685,28 +685,28 @@ struct Peep : SpriteBase
...
-        uint8_t guest_heading_to_ride_id;
+        uint8_t StaffId;
+        uint8_t GuestHeadingToRideId;
     };
     union
     {
-        uint8_t staff_orders;
-        uint8_t peep_is_lost_countdown;
+        uint8_t StaffOrders;
+        uint8_t PeepIsLostCountdown;
```
[the remark this answers is not in the document]
Comment (tupaschoal) [reply src/openrct2/peep/Peep.h:697]: You mean GuestIsLostCountdown?

File: src/openrct2/peep/Peep.h [hunk 2]
```diff
@@ -685,28 +685,28 @@ struct Peep : SpriteBase
...
+    rct12_xyzd8 PathfindHistory[4];
+    uint8_t NoActionFrameNum;
     // 0x3F Litter Count split into lots of 3 with time, 0xC0 Time since last recalc
-    uint8_t litter_count;
+    uint8_t LitterCount;
     union
     {
-        uint8_t time_on_ride;
-        uint8_t staff_mowing_timeout;
+        uint8_t TimeOnRide;
```
[the remark this answers is not in the document]
Comment (tupaschoal) [reply src/openrct2/peep/Peep.h:708]: GuestTimeOnRide?

File: src/openrct2/peep/Peep.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (duncanspumpkin) [reply src/openrct2/peep/Peep.h:697]: yes

File: src/openrct2/peep/Peep.h (hunk 2 above)
[the remark this answers is not in the document]
Comment (duncanspumpkin) [reply src/openrct2/peep/Peep.h:708]: yes

**Comment by Gymnasiast** [comment]
I'd rename `NoActionFrameNum` to `ActionFrameNum`.

**Comment by duncanspumpkin** [comment]
@Gymnasiast there is already an ActionFrameNum (`action_frame`). I think a better name for this variable would be `AnimationFrameNum`

**Comment by duncanspumpkin** [comment]
Argh that also doesn't quite work due to action_frame also being tied to animations. Its specifically a variable for the animation frame when the action is NONE_1 or NONE_2. Would `WalkingFrameNum` make more sense?

**Comment by Gymnasiast** [comment]
@duncanspumpkin It would at least make more sense than what we have currently.

**Comment by tupaschoal** [comment]
Did the renames

**Comment by Gymnasiast** [comment]
Last nitpick: could you reword ` Rename Peep::guest_heading_to_ride_id to use CamelCase` to ` Rename Peep::guest_heading_to_ride_id to GuestHeadingToRideId`, for correctness and consistency with the rest?

**Pull request merged**